### ISSUE TITLE
Bug: `iife` should startsWith `var` instead of `const` evev if i set preset to es2015

### ISSUE THREAD

**Issue by Jackie1210** [rollup/rollup#4585]
### Rollup Version

2.77.0

### Operating System (or Browser)

macOS

### Node Version (if applicable)

16.13.0

### Link To Reproduction

https://stackblitz.com/edit/rollup-template-83kwfj?file=rollup.config.js

### Expected Behaviour

```js
var lib = (function (exports) {
	'use strict';

	// run `node index.js` in the terminal

	const a = 'a';

	exports.a = a;

	Object.defineProperties(exports, { __esModule: { value: true }, [Symbol.toStringTag]: { value: 'Module' } });

	return exports;

})({});

```

### Actual Behaviour

```js
const lib = (function (exports) {
	'use strict';

	// run `node index.js` in the terminal

	const a = 'a';

	exports.a = a;

	Object.defineProperties(exports, { __esModule: { value: true }, [Symbol.toStringTag]: { value: 'Module' } });

	return exports;

})({});
```

**Comment by lukastaegert** [comment]
What is the bug you experience?

**Comment by Jackie1210** [comment]
@lukastaegert yep，you can check this.

https://github.com/vitejs/vite/issues/9318

**Comment by lukastaegert** [comment]
Fix at rollup/rollup#4588
